eng/versioning/version_client.txt

```diff
@@ -354,6 +354,7 @@ com.azure.tools:azure-sdk-build-tool;1.0.0-beta.1;1.0.0-beta.2
 unreleased_com.azure:azure-aot-graalvm-support;1.0.0-beta.1
 unreleased_com.azure:azure-aot-graalvm-support-netty;1.0.0-beta.1
 unreleased_com.azure:azure-aot-graalvm-perf;1.0.0-beta.1
+unreleased_com.azure:azure-core-amqp;2.5.0-beta.1
```

Comment:
Hmm interesting.. did we miss adding this post release?

Reply:
Oh after looking at the rest of the PR, looks like the version bumpis going in as part of this PR?

Reply:
Right, we want to ensure the core-lib change is not breaking EH and SB libs.